test/dataset/test_vad_dataset.py

```diff
@@ -0,0 +1,41 @@
+from math import isclose
+import pytest
+
+import torch
+
+from lhotse.utils import time_diff_to_num_frames
+from lhotse.cut import CutSet
+from lhotse.dataset.vad import VadDataset
+
+
+@pytest.fixture
+def cut_set():
+    return CutSet.from_yaml('test/fixtures/ami/cuts.yml')
+
+
+def test_unsupervised_dataset(cut_set):
+    duration = 5.0
+    feature_threhold = -8.0
+
+    cuts = list(cut_set.cuts.values())
```

Comment:
```suggestion
    cuts = list(cut_set)
```
Just syntactic sugar - it's possible because `CutSet` defines `__iter__` that returns `.cuts.values()`